### ISSUE TITLE
loss explosion when using 8 gpus

### ISSUE THREAD

**Issue by Oliver-ss** [feiyuhuahuo/Yolact_minimal#20]
It seems something wrong with the lr setting when using multiple gpus. 
As the reduction method of the loss function is sum. If the lr also multiplies by the number of gpus, the loss would immediately explode.

**Comment by feiyuhuahuo** [comment]
Hi, as far as I know, gradients are averaged across GPUs when training with multiple GPUs with the PyTorch framework. But there do has a bug, please set `self.warmup_until` directly as 500. I haven't fixed this yet. If loss still explodes please let me know.

**Comment by Oliver-ss** [comment]
```
def forward(self, backbone_outs):
        out = []
        x = torch.zeros(1, device=backbone_outs[0].device)
        for i in range(len(backbone_outs)):
            out.append(x)

        # For backward compatability, the conv layers are stored in reverse but the input and output is
        # given in the correct order. Thus, use j=-i-1 for the input and output and i for the conv layers.
        j = len(backbone_outs)  # convouts: C3, C4, C5

        for lat_layer in self.lat_layers:
            j -= 1

            if j < len(backbone_outs) - 1:
                _, _, h, w = backbone_outs[j].size()
                x = F.interpolate(x, size=(h, w), mode='bilinear', align_corners=False)

            x = x + lat_layer(backbone_outs[j])

            out[j] = x
```
The fpn module seems also have a bug. 
``` 
x = F.interpolate(x, size=(h, w), mode='bilinear', align_corners=False)
```
The variable x here seems always be 0.

**Comment by feiyuhuahuo** [comment]
![图片](https://user-images.githubusercontent.com/32631344/103610627-8f50fd80-4f5b-11eb-9ee6-09f015f6fae4.png)
I did not get 0. `x` should not be 0 because of `x = x + lat_layer(backbone_outs[j])`

**Comment by Oliver-ss** [comment]
I see, thx!
